definitions/checks/non_rh_packages.rb

```diff
@@ -27,8 +26,4 @@ def rh_regexp_list
      /-qpid-router-server/, /java-client/, /pulp-client/, /katello-default-ca/, /katello-server-ca/,
      /katello-ca-consumer/, /gpg-pubkey/, /-tomcat/]
```

Comment:
A lot of these are the certificate packages. I'd love to get rid of those as well. Is there anything tracking the complete removal of those? I'm mostly wondering when we can drop it due to the whole n-1 support.